aspnetcore/signalr/configuration.md

```diff
@@ -32,6 +33,36 @@ In the .NET client, the same `AddJsonProtocol` extension method exists on [HubCo
 // At the top of the file:
 using Microsoft.Extensions.DependencyInjection;
 
+// When constructing your connection:
+var connection = new HubConnectionBuilder()
+    .AddJsonProtocol(options => {
+        options.PayloadSerializerOptions.PropertyNamingPolicy = null;
+    })
+    .Build();
+```
+
+::: moniker-end
+
+::: moniker range="= aspnetcore-2.2"
```

Comment:
I suspect this content should display for both 2.1 and 2.2 (since this doc is available for 2.1 and later). If that's the case, make the following change:

```suggestion
::: moniker range="<= aspnetcore-2.2"
```